src/config/contributors.yaml

```diff
@@ -904,9 +904,9 @@ contributors:
     address: 10055 Barnes Canyon Road Suite 100 San Diego, CA 92121
     url: https://www.elementbiosciences.com/
     image_fn: elembio.svg
-    contact: Edmund Miller
-    contact_email: edmund.miller@elembio.com
-    contact_github: emiller88
+    contact: Bryan Lajoie
+    contact_email: https://www.elementbiosciences.com/contact
```

Comment:
This is prefixed with `mailto:` so a regular link won't work. Better to just remove.

```suggestion
```